source/lib/notifications-service.js

```diff
@@ -122,8 +124,18 @@ export async function playNotificationSound() {
 }
 
 export async function checkNotifications(lastModified) {
-	const notifications = await getNotifications({lastModified});
-	const {showDesktopNotif, playNotifSound} = await optionsStorage.getAll();
+	let notifications = await getNotifications({lastModified});
+	const {showDesktopNotif, playNotifSound, filterNotifications} = await optionsStorage.getAll();
+
+	if (filterNotifications) {
+		const repositories = await repositoriesStorage.getAll();
+		/* eslint-disable camelcase */
+		notifications = notifications.filter(({repository: {full_name}}) => {
+			const {owner, repository} = parseFullName(full_name);
+			return repositories[owner][repository];
```

Comment:
This line is throwing an error on me, when I have a notification to be displayed from `fregante/webext-options-sync`.

![image](https://user-images.githubusercontent.com/37769974/69705546-d2675d00-111b-11ea-8d5d-3605ef61856c.png)

I watch `fregante/webext-options-sync` but do not have access to it, so it would not show up when fetched with `/user/repos`. The same goes for other repos I don't have access to.